app/main.py

```diff
@@ -1,2 +1,102 @@
+from typing import Any
+
+
+class Node:
+    def __init__(self, key: Any, value: Any, next_node: Any = None) -> None:
+        self.key = key
+        self.value = value
+        self.next = next_node
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self, initial_capacity: int = 8,
```

Comment:
These attrs shouldnt be parametrized - it's part of dict internal structure. Also use Fraction for load factor